src/BlaspServiceProvider.php

```
@@ -0,0 +1,96 @@
+        $this->mergeConfigFrom(__DIR__ . '/../config/blasp.php', 'blasp');
+
+        $this->app->singleton('blasp', function ($app) {
+            return new BlaspManager($app);
+        });
+
+        $this->app->alias('blasp', BlaspManager::class);
+    }
+
+    protected function registerValidationRule(): void
+    {
+        $this->app['validator']->extend('blasp_check', function ($attribute, $value, $parameters) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attribute'. (undefined)

(UnusedFormalParameter)